scripts/variantstore/beta_docs/gvs-troubleshooting.md

```diff
@@ -8,19 +8,36 @@ Generally, if you have started the GVS workflow and it failed after ingestion, o
 2. `BadRequestException: 400 Bucket is a requester pays bucket but no user project provided.`
    1. GVS can ingest data from a requester pays bucket by setting the optional `billing_project_id` input variable. This variable takes a string of a Google project ID to charge for the egress of the GVCFs and index files.
 
-## Runtime errors
+## Ingestion-Specific Issues"
```

Comment:
```suggestion
### Ingestion-Specific Issues
```

I think you'd want this to be three headers in and the " seems to be a typo

Reply:
should they all be 3 headers in then?

Reply:
Sorry, I misread this and thought it was a subheader under ##Runtime errors.. I think that actually is how it should be. 

##Runtime errors
### Ingestion errors
[insert one sentence or so explanation of which tasks are 'ingestion' here]
1. error one 
2. error two

### Troubleshooting other pipeline failures
1. Other errors in document
2.
3.